packages/nc-gui/utils/filterUtils.ts

```diff
@@ -1,9 +1,8 @@
 import { UITypes, isNumericCol, numericUITypes } from 'nocodb-sdk'
 import { getI18n } from '../plugins/a.i18n'
 
-const getTranslation = (value: string) => getI18n().global.t(value)
-
 const getEqText = (fieldUiType: UITypes) => {
+  const { t } = getI18n().global
```

Comment:
you may just move it outside.

Reply:
This causes the app to crash, so I just followed same convention in `packages/nc-gui/utils/validation.ts`

<img width="1435" alt="Screenshot 2023-10-02 at 12 52 45" src="https://github.com/nocodb/nocodb/assets/1501599/e4e08d7c-3357-4823-aa25-876ff57e35b7">